packages/@react-spectrum/utils/src/Slots.tsx

```diff
@@ -33,11 +33,12 @@ export function cssModuleToSlots(cssModule) {
 }
 
 export function SlotProvider(props) {
-  let parentSlots = useContext(SlotContext) || {};
+  let stableEmptyObject = useRef({});
+  let parentSlots = useContext(SlotContext) || stableEmptyObject.current;
```

Comment:
Mind giving me the rundown on this change? Any particular reason for going with this approach rather than moving parentSlots into the useMemo below?

Reply:
you mean like this instead?
```
  let parentSlots = useContext(SlotContext);
  let {slots = {}, children} = props;

  // Merge props for each slot from parent context and props
  let value = useMemo(() =>
    Object.keys(parentSlots || {})
      .concat(Object.keys(slots))
      .reduce((o, p) => ({
        ...o,
        [p]: mergeProps(parentSlots[p] || {}, slots[p] || {})}), {})
      , [parentSlots, slots]);
```

I'm fine doing that too, it's probably slightly better

Reply:
so I tried that, and broke like every test... I'm not sure what went wrong, so sticking with stableEmptyObject for the moment

Reply:
Why is stableEmptyObject used instead of {}? things like that make me nervous that something more important is wrong.